scripts/globals/conquest.lua

```diff
@@ -95,7 +95,8 @@ BastInv = {0x80A1,0x000A,0x1055,0x80A0,0x0007,0x1056,0x80A3,0x2328,0x3CB5,
            0x8062,0x7D00,0x34F6,0x8060,0x7D00,0x3E55,0x8061,0x7D00,0x458F,
            0x8072,0x9C40,0x3355,0x8071,0x9C40,0x3638,0x8070,0x9C40,0x36BF,
            0x8080,0xBB80,0x419F,0x8081,0xBB80,0x4431,0x8083,0xBB80,0x44F7,
-           0x8082,0xBB80,0x4714,0x8090,0xDAC0,0x385D,0x80A4,0x1388,0x44B0};
+           0x8082,0xBB80,0x4714,0x8090,0xDAC0,0x385D,0x80A4,0x1388,0x44B0,
+		   0x80A8,0x1388,0x6F7C};
```

Comment:
It called an amend. But now you already have 2 commits so it'll take a squash/rebase operation.

To avoid similar tab mishaps in the future in notepad++ got to **`view`** then **`show symbol`** and then **`show whitespace and tab`** and they will always be visible.